### ISSUE TITLE
Image Style Filter The sci-fi filter effect of this node has a problem with the output image。

### ISSUE THREAD

**Issue by klossm** [WASasquatch/was-node-suite-comfyui#161]
Image Style Filter The sci-fi filter effect of this node has a problem with the output image, which is striped in black and white.

**Comment by WASasquatch** [comment]
> Image Style Filter The sci-fi filter effect of this node has a problem with the output image, which is striped in black and white.

Yeah I don't think that's supposed to be it's effect lol thanks. Can't remember if that's mine or pilgram. Will look into it.

**Comment by klossm** [comment]
I've checked and I don't think it's the original effect of the pilgram.I think it's yours.
